subsys/bluetooth/host/df.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_BLUETOOTH_HOST_DF_H_
+#define ZEPHYR_SUBSYS_BLUETOOTH_HOST_DF_H_
+
+#include <sys/util.h>
+
+enum df_opt_swich_sampl_rates {
+	/* Support Angle of Departure 1us switching TX */
+	DF_AOD_1US_SWITCH_TX_SUPP = BIT(0),
+	/* Support Angle of departure 1us sampling RX */
+	DF_AOD_1US_SAMPL_RX_SUPP = BIT(1),
+	/* Support Angle of Arrival 1us switching and sampling RX */
+	DF_AOA_1US_SWITCH_SAMPL_RX_SUPP  = BIT(2),
+};
+
+/* @brief Antenna information for LE Direction Finding */
+struct bt_le_df_ant_inf {
+	uint8_t switch_sampl_rates;
+	uint8_t num_ant;
+	uint8_t max_switch_pattern_len;
+	uint8_t max_CTE_len;
+};
```

Comment:
Why not reuse HCI info structure?

Reply:
I thought that types declared in hci.h are used in HCI related source code.

The aim of this declaration is to provide a type to use it in implementation of higher level code. E.g. in future if DF related services or profiles are defined. Right now this part of code is used only by test code and may use HCI info structure as well. Question here is about current approach for such situations. Is it expected to have a separation of types between "layers" or reuse types as much as possible?